test/data/cr_things_filter.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+apiVersion: stable.nicolerenee.io/v1
+kind: Character
+metadata:
+  name: thing1
+  annotations:
+    optIn: Opting In
```

Comment:
Missed this before. Can we update this example with a full DNS example so that it is valid. Maybe here use `io.nicolerenee.lostromosApplied` or something not under the `com.wpengine.lostromos` so that people will see it can be any value. But it should be DNS